pkg/controller/schedule_controller.go

```diff
@@ -199,11 +199,10 @@ func (c *scheduleReconciler) checkIfBackupInNewOrProgress(schedule *velerov1.Sch
 
 	for _, backup := range backupList.Items {
 		if backup.Status.Phase == velerov1.BackupPhaseNew || backup.Status.Phase == velerov1.BackupPhaseInProgress {
+			log.Debugf("%s/%s still has backups that are in InProgress...", schedule.Namespace, schedule.Name)
```

Comment:
Change it to `in InProgress or New`? That's confuse if only contains the `InProgress` state

Reply:
Got it.

Reply:
Rebased, Thanks review.